refinery/ui/source/js/file-browser/views/files-tab.html

```diff
@@ -24,9 +28,28 @@
           </span>
         </button>
 
-        <span class="float-right">
-          <span ng-if="FBCtrl.isOwner" class="refinery-base btn btn-default btn-sm" rp-assay-files-util-modal>
+        <span class="float-right content">
+          <span
+            ng-if="FBCtrl.isOwner"
+            class="refinery-base btn btn-default btn-sm"
+            ng-click="FBCtrl.openPermissionEditor()"
+            refinery-tooltip
+            refinery-tooltip-container="body"
+            refinery-tooltip-placement="bottom"
+            title="Launch modal to update the data set's permissions">
```

Comment:
Here and below, user doesn't care so much about the machinery... would "Update the data set's permissions" suffice?